src/System.Private.CoreLib/shared/Interop/Unix/System.Native/Interop.SysLog.cs

```diff
@@ -54,6 +54,6 @@ internal enum SysLogPriority : int
         /// <param name="message">The message to put in the log entry</param>
         /// <param name="arg1">Like printf, the argument is passed to the variadic part of the C++ function to wildcards in the message</param>
         [DllImport(Libraries.SystemNative, EntryPoint = "SystemNative_SysLog")]
-        internal static extern void SysLog(SysLogPriority priority, string message, string arg1);
+        internal static extern void SysLog(SysLogPriority priority, string message, string? arg1);
```

Comment:
In what situation could a `null` make it to `arg1`?  That would end up effectively passing a `null` pointer into `syslog` as a format argument, and I don't believe it would take kindly to that.